doc/source/categorical.rst

```diff
@@ -189,7 +253,9 @@ It's also possible to pass in the categories in a specific order:
 
     .. ipython:: python
 
-         s = pd.Series(list('babc')).astype('category', categories=list('abcd'))
+         s = pd.Series(list('babc')).astype(
+             pd.api.types.CategoricalDtype(list('abcd'))
```

Comment:
I would import ``CategoricalDtype`` at the top of ``categorical.rst`` (or in an appropriate place) I think.